tests/ArraySize/yosys_script

```diff
@@ -1,5 +1,5 @@
 plugin -i uhdm
-read_uhdm -debug top.uhdm
+read_verilog_with_uhdm -parse ../tests/ArraySize/top.sv
```

Comment:
I'd say we shouldn't replace current used frontend, but instead set some environment variable in Makefile, and then use it here to detect what frontend we want to use.

In case of ``read_verilog_with_uhdm`` we shouldn't call Surelog to generate UHDM file.

Reply:
I don't know how to pass the environment variable to yosys_scipt. I see only the `exec` option, but it can be used to run shell commands. Here I want to run yosys command with shell variable that needs to be evaluated.
Do you have any ideas?

Reply:
We can use tcl script for this. From yosys help:
```
    -c tcl_scriptfile
        execute the commands in the tcl script file (see 'help tcl' for details)
...
yosys> help tcl

    tcl <filename> [args]

This command executes the tcl commands in the specified file.
Use 'yosys cmd' to run the yosys command 'cmd' from tcl.

The tcl command 'yosys -import' can be used to import all yosys
commands directly as tcl commands to the tcl shell. Yosys commands
'proc' and 'rename' are wrapped to tcl commands 'procs' and 'renames'
in order to avoid a name collision with the built in commands.

If any arguments are specified, these arguments are provided to the script via
the standard $argc and $argv variables.
```

Reply:
Thank you. I used it.